logging/log_config_test.go

```diff
@@ -176,18 +121,6 @@ func TestUpdateLoggerRegistry(t *testing.T) {
 				"rdk.resource_manager.package_manager.modmanager": "WARN",
 			},
 		},
-		{
-			loggerConfig: []LoggerPatternConfig{
-				{
-					Pattern: "_.*.modmanager",
-					Level:   "DEBUG",
-				},
-			},
-			loggerNames: []string{
-				"rdk.resource_manager",
-			},
-			expectedMatches: map[string]string{},
```

Comment:
This test made no assertions. I fear maybe this test was for some special `_` character that maybe meant `*` but for only one sublogger?

Rather than leave future readers guessing, I just removed the test.